### PR TITLE
Document the envisioned content approach and ownership model

### PR DESCRIPTION
**Pull request by PeterJCLaw**
This documents what was originally envisioned for the runbook.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -33,6 +33,35 @@ There are many cases where it should not contain the actual documentation:
...
+newcomers, for both reading and contributing.
+
+It's worth noting that, as of early 2020, the runbook is still a way off
+achieving this aim as it contains considerable left-over structure & content
+from the previous documentation which it subsumed.
+
+## Ownership
+
+While the runbook contains documentation relevant to the activities of the
+Competition and Kit Teams, neither Team owns it any more than any other
```
Comment (trickeydan) [inline docs/README.md:54]: More teams now exist. I know this is intended as the original model, so feel free to ignore. 

```suggestion
all SR teams, no one team owns it any more than any other
```

Comment (PeterJCLaw) [reply docs/README.md:54]: Ah, good spot. This was written a long while back and I should have self-reviewed it!

File: docs/README.md [hunk 2]
```diff
@@ -33,6 +33,35 @@ There are many cases where it should not contain the actual documentation:
...
+newcomers, for both reading and contributing.
+
+It's worth noting that, as of late 2021, the runbook is still a way off
+achieving this aim as it contains considerable left-over structure & content
+from the previous documentation which it subsumed.
+
+## Ownership
+
+While the runbook contains documentation relevant to the activities of the
+all SR teams, no one team owns it any more than any other
```
Comment (RealOrangeOne) [inline docs/README.md:54]: Issue with the 4a7a822 suggestion

```suggestion
While the runbook contains documentation relevant to the activities of
all SR teams, no one team owns it any more than any other
```

Comment (PeterJCLaw) [reply docs/README.md:54]: :facepalm:

**Review by trickeydan — APPROVED** [review]
I believe this accurately reflects the original aims of the runbook 🎉

**Pull request merged**